### PR TITLE
add isolate elements feature to transformer test app

### PR DESCRIPTION
**Pull request by MichaelBelousov**
ever since the removal of deferring from the transformer, the transformer will now export all *required* (just the parent/model and category) predecessors recursively when asked to export an element. This means it's now trivial to create a filtered iModel from an element id, containing only the element and its subtree of children.

Now there is a command line flag in the imodel-transformer test app to do this. This makes trimming an imodel to send to someone else much easier. It is especially great for development, when reducing the size of an iModel containing elements that reproduce particular behavior.

I also added some missing geometry transformer options flags to the test-app's CLI

### CODE REVIEW

File: test-apps/imodel-transformer/src/Main.ts [hunk 1]
```diff
@@ -165,6 +165,20 @@ void (async () => {
...
+          desc: "transform filtering all unnecessary element/model trees except for those listed in a comma-separated argument of ids",
+          type: "string",
+        },
+        loadSourceGeometry: {
+          desc: "load geometry from the source as JSON while transforming, for easier (but not performant) transforming of geometry",
+          type: "boolean",
+          default: false,
+        },
+        cloneUsingJsonGeometry: {
+          desc: "clone using json geometry",
```
Comment (ColinKerr) [inline test-apps/imodel-transformer/src/Main.ts:178]: can you give more helpful info in the description, this just repeats the parameter name with spaces

Comment (MichaelBelousov) [reply test-apps/imodel-transformer/src/Main.ts:178]: whoops I meant to use a better description but wasn't sure if I was going to keep this one. Will fix.

Comment (MichaelBelousov) [reply test-apps/imodel-transformer/src/Main.ts]: is that better?

**Review by ColinKerr — APPROVED** [review]

**Review by pmconne — COMMENTED** [review]
NextVersion.md?

**Comment by MichaelBelousov** [comment]
> NextVersion.md?

This is an internal tool. Does it even count as part of the new version? You need to build it yourself too

**Comment by pmconne** [comment]
> This is an internal tool/test-app

My mistake - I thought it was a change to core-transformer.

**Comment by MichaelBelousov** [comment]
@mergify update

**Comment by MichaelBelousov** [comment]
since this is still failing due to flaky tests, I had another flag I was planning on adding in a separate PR, but I'll add it to this PR and will re-request review @ColinKerr

**Comment by MichaelBelousov** [comment]
@ColinKerr I added `--isolateTrees` separate from `--isolateElements`, now `--isolateElements` will not continue to transform any children of the isolated elements, but `--isolateTrees` exists for that use case

**Review by ColinKerr — APPROVED** [review]

**Pull request merged**